app/services/licences/supplementary-billing-years.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+/**
+ * Works out which financial year ends are between the date ranges
+ * @module SupplementaryBillingYearsService
+ */
+
+const MONTHS_UNTIL_FINANCIAL_YEAR_START = 3
```

Comment:
It's my own personal opinion. But we're testing if the month of a date is equal to or greater than April (assuming I'm not mistaken).

So, if we renamed this to `APRIL` do you think it would make where we use it a little clearer?

```javascript
  if (date.getMonth() >= APRIL) {
    // ..
  }
```